### PR TITLE
Add meaningful REST API endpoint descriptions

### PR DESCRIPTION
**Pull request by quba42**
fixes #7355
https://pulp.plan.io/issues/7355

Motivated by https://github.com/pulp/pulp_deb/pull/198

The existing REST API docs (for reference): https://pulp-deb.readthedocs.io/en/latest/restapi.html

### CODE REVIEW

**Comment by quba42** [comment]
`flake8` is demanding both that doc strings start with a single line ending in a period, followed by a blank line, AND that lines are no longer than 100 characters.

These two rules combined essentially make it impossible, to have a meaningful first line in doc strings for classes involving complex concepts.

I think at least one of these `flake8` rules should be dropped, but I am unsure what a good style convention would be.
Does anyone have any suggestions?

**Comment by quba42** [comment]
`flake8` is demanding both that doc strings start with a single line ending in a period, followed by a blank line, AND that lines are no longer than 100 characters.

These two rules combined essentially make it impossible, to have a meaningful first line in doc strings for classes involving complex concepts.

I think at least one of these `flake8` rules should be dropped, but I am unsure what a good style convention would be.
Does anyone have any suggestions?

File: pulp_deb/app/viewsets/content.py [hunk 1]
```diff
@@ -60,8 +66,11 @@ class Meta:
 
 
 class PackageViewSet(SingleArtifactContentUploadViewSet):
+    # The doc string is a top level element of the user facing REST API documentation:
```
Comment (mdellweg) [inline pulp_deb/app/viewsets/content.py:69]: Nice catch!

Comment (quba42) [reply pulp_deb/app/viewsets/content.py:69]: First, I wanted to write about each content type in the documentations "Feature Overview" chapter. Then I noticed that I was basically writing a second REST API documentation, so I went and had a look at the actual REST API documentation, and noticed all the descriptions were "A ViewSet for <class_name>.".

So it did take a couple of steps to get  there. :stuck_out_tongue_winking_eye:

**Pull request merged**